data/RWTH-NFDI-Jupyter-Service.yaml

```diff
@@ -0,0 +1,42 @@
+title: RWTH Aachen University
+provider: HPC.ITC.RWTH
+service_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/
+support: servicedesk@itc.rwth-aachen.de
+health_api_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/hub/api/health
+documentation_url: https://help.itc.rwth-aachen.de/service/rhr4fjjutttf/article/689934fec5a34c909c54606f6bc2e827/
+target_group_open_for: Researchers, RWTH Students/Employees/Guests, Juelich Members, Users sponsored by other Users, Users upon project application 
+login_process: Login with RWTH HPC Account through most NHR or NRW VPNs
+features:
+  version: JupyterHub 3.0+, JupyterLab 3.4.7+
+  kernels: ["Python 3.9", "Octave", "R", "C++ 11/14/17", "Matlab 2022b", "Matlab 2020b",  "Python: Keras", "Python: Tensorflow",  "Python: Tensorboard", "Python: Gurobi", "Python: Pytorch" ]
```

Comment:
- `"Matlab 2022b", "Matlab 2020b"` do you mean the official Mathworks kernels or the one from Calysto? https://github.com/calysto/matlab_kernel
- `"Python: Keras", "Python: Tensorflow",  "Python: Tensorboard", "Python: Gurobi", "Python: Pytorch"` Are these entries really kernels ( https://docs.jupyter.org/en/latest/projects/kernels.html ), or are these installed libs? We have not a field for installed libs in the page yet - would you be interested in such?

Reply:
- The matlab as listed there are Both Kernels and Proxies from Mathworks.
- The Pythons are Python Kernels with installed libraries. I don't think an extra special field is necessary. Our system works with pre-installed "profiles" where a Kernel has various libraries already installed. The listing here was my attempt to map our profiles to this yaml accordingly.

Reply:
Ah, this are virtual environments. I also found some [documentation](https://help.itc.rwth-aachen.de/service/rhr4fjjutttf/article/50a810bf34da41c3a37bd315505af92a/), where actually are even more profiles that you have listed here. Is this the ground truth?

Reply:
Yes, they are conda environments with pre-chosen packages.